dasharo-compatibility/dcu.robot

```diff
@@ -0,0 +1,160 @@
+# - go threough them and make sure they are doing what the name suggest (not
+# exactly the case right now)
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+# ...    AND
+# ...    Prepare DCU Test Environment
+# ...    AND
+# ...    Make Sure That Flash Locks Are Disabled
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+DCU001.001 Change the UUID
+    [Documentation]    This test case verifies that the UUID encoded in the DMI
+    ...    table of an image can be changed using DCU.
+    Skip If    not ${UUID_SETTABLE}    DCU001.001 not supported
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DCU001.001 not supported
+    Skip
+    ${uuid}=    Set Variable    96bcfa1a-42b4-6717-a44c-d8bbc18cbea4
+    ${result}=    Run Process    bash    -c    ./dcu/dcu smbios -u ${uuid} ./dcu/coreboot.rom
+    Should Contain    ${result.stdout}    Success
+    Flash Firmware    ./dcu/coreboot.rom
+
+    Power On
+    Boot Operating System    ubuntu
+    Login To Linux
+    Switch To Root User
+    ${uuid_read}=    Get Firmware UUID
+    Should Be Equal As Strings    '${uuid}'    '${uuid_read}'
+
+DCU002.001 Change the serial number
+    [Documentation]    This test case verifies that the serial number encoded
+    ...    in the DMI table of an image can be changed using DCU.
+    Skip If    not ${SERIAL_NUMBER_SETTABLE}    DCU002.001 not supported
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DCU002.001 not supported
+    Skip
+    ${serial_no}=    Set Variable    D01234567
+    ${result}=    Run Process    bash    -c    ./dcu/dcu smbios -s ${serial_no} ./dcu/coreboot.rom
+    Should Contain    ${result.stdout}    Success
+    Flash Firmware    ./dcu/coreboot.rom
+
+    Power On
+    Boot Operating System    ubuntu
+    Login To Linux
+    Switch To Root User
+    ${serial_no_read}=    Get Firmware Serial Number
+    Should Be Equal As Strings    '${serial_no}'    '${serial_no_read}'
+
+DCU003.001 Change the bootsplash logo
+    [Documentation]    This test case verifies that the bootsplash logo encoded
+    ...    into an image can be changed using DCU.
+    ...    PLEASE NOTE that a display device needs to be physically connected
+    ...    to the DUT for this test to work.
+    Skip If    not ${BOOTSPLASH_SETTABLE}    DCU003.001 not supported
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DCU003.001 not supported
+    Skip
+    ${img_sum}=    Set Variable    f91fe017bef1f98ce292bde1c2c7c61edf7b51e9c96d25c33bfac90f50de4513
+    ${result}=    Run Process    bash    -c    ./dcu/dcu logo -l ./dcu/logo.bmp ./dcu/coreboot.rom
+    Should Contain    ${result.stdout}    Success
+    Flash Firmware    ./dcu/coreboot.rom
+
+    Power On
+    Boot Operating System    ubuntu
+    Set Global Variable    ${DUT_CONNECTION_METHOD}    SSH
+    Login To Linux
+    Switch To Root User
+    Write Into Terminal    sha256sum /sys/firmware/acpi/bgrt/image
```

Comment:
Why not Execute Command In Terminal ?

Reply:
[Changed](https://github.com/Dasharo/open-source-firmware-validation/pull/139/files#diff-7ee5e21e59cd627383812396fcf45a6e7ecb3e2f096ab5ae5203a0c6bd75989fR88) to Execute Command In Terminal